### PR TITLE
feat: update release notes

### PR DESCRIPTION
**Pull request by daudix-UFO**
# 0.3.2 release notes

Description of what changed in this release

## Changelog

- Added 0.3.2 release notes

## Screenshot

If applicable

### CODE REVIEW

**Comment by LyesSaadi** [comment]
Also, changing all these to the bottow ones as well !

**Comment by daudix-UFO** [comment]
Wealso need to update small release notes in welcome window

**Comment by daudix-UFO** [comment]
> In this release, we fixed Firefox GNOME theme plugin, issue when presets always saved using one name, as well as some UX polish, and a lot more.

**Comment by daudix-UFO** [comment]
Is this OK? ^

**Comment by LyesSaadi** [comment]
How about :

> In this release, we fixed the Firefox GNOME theme plugin, issues with presets always being saved with the same name, as well as some UX polish, and more.

**Comment by daudix-UFO** [comment]
Like it :+1:

**Comment by daudix-UFO** [comment]
image

**Comment by daudix-UFO** [comment]
@LyesSaadi

**Comment by LyesSaadi** [comment]
Btw, @daudix-UFO, did you see my reviews ?

**Comment by daudix-UFO** [comment]
Umm, no :/

**Comment by daudix-UFO** [comment]
image

**Comment by LyesSaadi** [comment]
uh, try refreshing the page ?

**Comment by LyesSaadi** [comment]
image

**Comment by daudix-UFO** [comment]
Stil nothing :/ maybe you not published it?

**Comment by LyesSaadi** [comment]
Did this do the trick ?

**Comment by daudix-UFO** [comment]
Yes!!!

**Comment by LyesSaadi** [comment]
Ah, sorry, still confused by github xD

**Comment by daudix-UFO** [comment]
Me too :///

**Comment by daudix-UFO** [comment]
Uh oh, looks like I messed up something

File: data/com.github.GradienceTeam.Gradience.appdata.xml.in.in [hunk 1]
```diff
@@ -38,9 +38,17 @@
...
+          <li>Save is now a default response in dialogs</li>
+          <li>Plugins rows now look cleaner and correctly placed</li>
+          <li>File picker is now modal and sticks with the parent window</li>
+          <li>Gradience now can be closed with Ctrl+Q</li>
+          <li>"Favourite(s)" was renamed to "Favorite(s)"</li>
           <li>Тransition from cssutils to a in-house solution</li>
           <li>Fixed issues with CSS parser</li>
           <li>Fixed issue when presets always saved as User.json</li>
+          <li>Presets now removes correctly</li>
           <li>Internal structure was refactored</li>
```
[the remark this answers is not in the document]
Comment (daudix-UFO) [reply data/com.github.GradienceTeam.Gradience.appdata.xml.in.in]: The presets… ?

**Comment by daudix-UFO** [comment]
Everything OK now?

File: data/com.github.GradienceTeam.Gradience.appdata.xml.in.in (hunk 1 above)
[the remark this answers is not in the document]
Comment (LyesSaadi) [reply data/com.github.GradienceTeam.Gradience.appdata.xml.in.in:52]: No, The internal structure

[the remark this answers is not in the document]
Comment (LyesSaadi) [reply data/com.github.GradienceTeam.Gradience.appdata.xml.in.in:52]: Like in the The update

File: data/com.github.GradienceTeam.Gradience.appdata.xml.in.in [hunk 2]
```diff
@@ -38,9 +38,17 @@
...
           <li>Various typos were fixed</li>
           <li>README was fully rewritten</li>
```
[the remark this answers is not in the document]
Comment (LyesSaadi) [reply data/com.github.GradienceTeam.Gradience.appdata.xml.in.in:54]: The README

**Comment by LyesSaadi** [comment]
> Everything OK now?

Uh no

**Comment by LyesSaadi** [comment]
Some minor nitpicking the the The Update, you misunderstood what I meant. I should've sent comments answering things ?

**Comment by LyesSaadi** [comment]
You know what, maybe let me commit those myself if it's easier :sweat_smile: ?

**Comment by daudix-UFO** [comment]
You were able to do this the whole time? you are monster

**Comment by daudix-UFO** [comment]
This is officially the most F'ed up PR in Gradience history :sweat_smile:

**Comment by LyesSaadi** [comment]
Oh, well, you did it xD !

**Comment by daudix-UFO** [comment]
"LGTM" ?

**Comment by LyesSaadi** [comment]
Just a minor nitpick before... Let me commit that myself xD.

**Comment by daudix-UFO** [comment]
Yes please

**Comment by LyesSaadi** [comment]
There xD ! Now I can say it :

LGTM

**Comment by daudix-UFO** [comment]
Merge time!

**Comment by daudix-UFO** [comment]
Well, now I have one small nitpick

**Pull request merged**